app/views/articles/index.html.erb

```diff
@@ -1,13 +1,46 @@
-<h1>Articles</h1>
-Our blog has <%= Article.public_count %>
-articles and counting!
-<ul>
-  <% @articles.each do |article| %>
-    <% unless article.archived? %>
-    <li>
-      <%= link_to article.title, article%>
-    </li>
-    <% end %>
-  <% end %>
-</ul>
-<%= link_to 'New Article', new_article_path %>
+
```

Comment:
Please review the indentation in this file.

Reply:
Done !